test/integration/retryable-writes/retryable_writes.spec.test.ts

```diff
@@ -14,6 +14,17 @@ interface RetryableWriteTestContext {
   failPointName?: any;
 }
 
+const ALLOWED_BULK_PROPERTIES = [
+  'acknowledged',
+  'insertedCount',
+  'matchedCount',
+  'modifiedCount',
+  'deletedCount',
+  'upsertedCount',
+  'upsertedIds',
+  'insertedIds'
```

Comment:
```suggestion
  'insertedIds',
  'upsertedId'
```

and maybe more - CI is pretty red.

Reply:
This is now removed as we have cases where actual documents are returned. See next comment.